src/loaders/markdown/transformer/remarkContainer.ts

```diff
@@ -24,22 +34,17 @@ export default function remarkContainer(this: any): Transformer<Root> {
 
   return (tree) => {
     visit<Root>(tree, (node, i, parent) => {
-      if (
-        node.type === 'containerDirective' &&
-        VALID_CONTAINER_TYPES.includes(node.name)
-      ) {
-        const attrs = Object.entries(node.attributes || {}).reduce<string>(
-          (ret, [name, value]) =>
-            `${ret} ${value ? `${name}="${value}"` : name}`,
-          '',
-        );
+      if (node.type !== 'containerDirective') return CONTINUE;
+
+      if (VALID_CONTAINER_TYPES.includes(node.name)) {
+        const attrs = transformAttributes(node.attributes);
         // replace directive node with container node
         parent!.children.splice(
           i!,
           1,
           {
             type: 'html',
-            value: `<Container type="${node.name}"${attrs}>`,
+            value: `<Container type="${node.name}" ${attrs}>`,
```

Comment:
`transformAttributes` 的执行结果一定有头部空格

```suggestion
            value: `<Container type="${node.name}"${attrs}>`,
```